### DISCUSSION TITLE
Cleanup after failed installation of a sensor

### DISCUSSION THREAD

**Discussion by firion4ik** [Security-Onion-Solutions/securityonion#1566] [Unsupported Versions]
Hello!

I would like to know if there is any guidance on how to clean up after failed installation (network type) attempt of a sensor?
We have a physical server for the sensor and it takes approx a whole day for our IT team and Network team (if I'm actively chasing them) to re-install the OS. 
Would be great to know which files/folders I can delete/edit on the sensor and manager node before I can try to install again to make sure new installation doesn't interfere with previous installation artifacts.
I searched the subReddit and Google groups for the same question but didn't find anything.

**Comment by weslambert** [comment]
We don't currently have a process documented, but we are working on it from a setup perspective, here: https://github.com/Security-Onion-Solutions/securityonion/issues/1411

**Comment by ghost** [comment]
+1 to this. I hope to see that documentation soon. I've got a problem leading me to need the same thing as the OP: A way to clean a sensor "profile" out.  It'll help a lot when you all are able to publish it. 

Thanks for the hard work SO Team!
